tools/DfmHttpService/DfmHttpServer.cs

```diff
@@ -35,6 +35,7 @@ public void Start()
 
             _listener.Start();
             _processing.Reset();
+            Console.WriteLine("Ready");
```

Comment:
Here, when the child process is ready to handle the request, it will give a console output which can be detected by extension. After that, all communication can be implemented by Http request. we can implement it in memory level, but it will be complicated.

---
In reply to: [108123505](https://github.com/dotnet/docfx/pull/1490#discussion_r108123505) [](ancestors = 108123505,108114901,108107907)